components/base/agilex/limo_base.go

```diff
@@ -53,7 +53,7 @@ func (m steeringMode) String() string {
 	return "Unknown"
 }
 
-var modelName = resource.NewDefaultModel("agilex-limo")
+var ModelName = resource.NewDefaultModel("agilex-limo")
```

Comment:
[q] is there any way we can keep these unexported? Or will that make it impossible to know which frames are attached to each base in fake?

Reply:
This is the pattern we settled on for arms, so I figured it would make sense to apply it to bases too.  Without exporting these we would be relying on hardcoding these strings in the fake package, which is a lot worse IMO.  What is your hesitation with exporting?

Reply:
Just that there are two patterns going on - one to unexport as much as we can from each package because that keeps things more under control for someone else importing these packages for their own code. It's not a blocker, just something to think about when deciding what to import/export.

Reply:
This could go away for non-wheeled bases if you are good with hardcoding fake as wheeled for now

Reply:
I am okay with this for now. Just be aware this might need to be re-tooled in the future, but we'll cross that bridge if/when we get there.